hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -64,9 +65,10 @@
     static {
         String loggingType = "hazelcast.logging.type";
         String loggingClass = "hazelcast.logging.class";
-        if (System.getProperty(loggingType) == null && System.getProperty(loggingClass) == null) {
+        if (JenkinsDetector.isOnJenkins()) {
```

Comment:
what about `JenkinsDetector.isOnJenkins() || (System.getProperty(loggingType) == null && System.getProperty(loggingClass) == null)` ?

I'd like to see same logging while running tests on my local, instead of defining logging type each time.